.chloggen/replace-all-continue.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: pkg/ottl
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Small behavior change in that when an error occurs will continue to the next values.
```

```suggestion
note: Keep the original map and continue processing keys if an error is encountered when calling an optional replacement function in `replace_all_patterns`.
```
I think this more accurately describes the exact change.